boards/x86/up_squared/dts_fixup.h

```diff
@@ -7,9 +7,153 @@
 /* Board level DTS fixup file */
 
 #ifdef CONFIG_SBL_FIXUP
```

Comment:
can we please put this in a separate commit?

Reply:
Sure, separate PR?

Reply:
no need for new PR

Reply:
Done